src/env.cc.in

```diff
@@ -146,11 +160,12 @@ const std::vector<std::string> Env::cyclus_path() {
   std::cout << __FILE__ << ":" << __LINE__ << ": "#X" = " << X << "\n"
 
 std::string Env::FindModule(std::string path) {
+  boost::system::error_code errc;
   std::vector<std::string> strs = cyclus_path();
 
   for (int i = 0; i < strs.size(); ++i) {
     fs::path full = fs::path(strs[i]) / path;
-    if (fs::exists(full)) {
+    if (fs::exists(full, errc)) {
```

Comment:
This was cyclus misusing the Boost API [1].  It turns out that if you pass in an error reference, the result status will be stored in the error code and it will return a boolean normally.  However, if you don't pass in an error code, then in the event of an invalid status (like when a file doesn't exist) then this function will also throw an error, effectively preventing the if statement from evaluating the conditional [2]. It is much safer to pass in the error code and is probably something we should always do with these functions.

1. http://www.boost.org/doc/libs/1_61_0/libs/filesystem/doc/reference.html#exists
2. http://www.boost.org/doc/libs/1_61_0/libs/filesystem/doc/reference.html#Error-reporting